src/Microsoft.DotNet.MSBuildSdkResolver/MSBuildSdkResolver.cs

```diff
@@ -139,17 +130,20 @@ public override SdkResult Resolve(SdkReference sdkReference, SdkResolverContext
             string msbuildSdkDir = Path.Combine(msbuildSdksDir, sdkReference.Name, "Sdk");
             if (!Directory.Exists(msbuildSdkDir))
             {
-                return factory.IndicateFailure(
-                    new[] 
-                    {
-                        $"{msbuildSdkDir} not found. Check that a recent enough .NET Core SDK is installed"
-                        + " and/or increase the version specified in global.json."
-                    });
+                return Failure(
+                    "{0} not found. Check that a recent enough .NET Core SDK is installed and/or increase the version specified in global.json.",
+                    factory,
+                    msbuildSdkDir);
             }
 
             return factory.IndicateSuccess(msbuildSdkDir, netcoreSdkVersion);
         }
 
+        private SdkResult Failure(string format, SdkResultFactory factory, params string[] args)
```

Comment:
Nit: perhaps the parameter order should be `(SdkResultFactory factory, string format, params string[] args)` since the args pertain to the format string?

Reply:
Yeah, I'll fix this.

Reply:
Also fixed.